unified-codes/apps/web/src/components/containers/error/ErrorNotFoundCenter.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+
+import { makeStyles, createStyles } from '@unified-codes/ui/styles';
+
+import { ITheme } from '../../../styles';
+
+const image = require('../../../assets/image/cat_bike.png');
+
+const useStyles = makeStyles((_: ITheme) => createStyles({
+    image: {
+        maxHeight: 'calc(100vh - 350px)', 
+        maxWidth: '100vw',
+        padding: '20px 0 20px 0'
+    },
+}));
+
+export interface ErrorNotFoundCenterProps {}
+
+export type ErrorNotFoundCenter = React.FunctionComponent<ErrorNotFoundCenterProps>;
+
+export const ErrorNotFoundCenter: ErrorNotFoundCenter = () => {
+    const classes = useStyles();
+    return <img className={classes.image} src={image} />;
+};
+
+export default ErrorNotFoundCenter;
```

Comment:
wait, what? `Centre` surely?

Reply:
I've been beaten down by the Americanization of everything browser/CSS related. Even if I revert back to `"center"`, the commit history is all that's needed for them to lock my up in the tower. 

@katherine-sussol @mark-prins Front-end spelling conventions: American English or correct English?

Reply:
🤣  that sounded like a loaded question!
I'm a pedant, and I run with `correct English` - really doesn't trouble me either way though

Reply:
I also prefer correct English! We mustn't give in to these external pressures to Americanize!
... Actually I don't mind either and tailor depending on audience. But I think our audience here is !🇺🇸